alpaca_trade_api/rest.py

```diff
@@ -884,6 +898,43 @@ def get_crypto_snapshot(self, symbol: str, exchange: str) -> SnapshotV2:
                              api_version='v1beta1')
         return self.response_wrapper(resp, SnapshotV2)
 
+    def get_news_iter(self,
+                      symbol: Union[str, List[str]],
+                      start: Optional[str] = None,
+                      end: Optional[str] = None,
+                      limit: int = None,
+                      sort: Sort = Sort.Desc,
+                      include_content: bool = False,
+                      exclude_contentless: bool = False,
+                      raw=False) -> NewsIterator:
+        if isinstance(symbol, str):
+            symbol = [symbol]
+        news = self._data_get('news', symbol,
```

Comment:
I think it makes slightly more sense to have the `endpoint_base='news'` and `endpoint=''`. Also, please use key arguments!